src/worker.ts

```diff
@@ -12,30 +12,27 @@ function markets(orderbooksString: string, base: string, quote: string, hops: nu
 }
 
 function estimatedBuyAmount(
-  orderbooksString: string,
-  base: string,
-  quote: string,
-  hops: number,
-  quoteAmount: string,
-  priceRoundingBuffer: number,
```

Comment:
I think we should keep this rounding buffer. Or is the pricegraph making sure that we are not running into rounding issues when the estimate is eventually passed into the solver?

Reply:
Hmm, that's a good question. I think for now, it is probably safer to leave this in - especially given how placing a trade with the price estimate and having it not matched is a very poor user experience.